src/plot_api/plot_api.js

```diff
@@ -42,6 +42,8 @@ var enforceAxisConstraints = axisConstraints.enforce;
 var cleanAxisConstraints = axisConstraints.clean;
 var axisIds = require('../plots/cartesian/axis_ids');
 
+var createRegl = require('regl');
```

Comment:
Ha. `regl` must be invoking one of many globals that IE9 doesn't support at runtime.

As this file here `plot_api.js` is required in all partial bundles (even that gl-less ones), this makes the IE9 test fail.

So, I think that best to solve this would be use a `Registry.getComponentMethod`-style pattern so that `regl` is only required in bundles that expose `regl` traces.

Reply:
@etpinard do I get it right that we don't have this safe-require pattern yet? What if I just require regl directly when the `regl` trace type is detected?

Reply:
>  What if I just require regl directly when the regl trace type is detected?

You mean something like:

```js
if(fullLayout._has('regl')) {
  var createREGL = require('regl')
}
```

in `plot_api.js`? That could work, `regl` would still got bundled up in all partial bundles.